src/modbus-tcp.c

```diff
@@ -41,7 +41,9 @@
 #endif
 
 # include <netinet/in.h>
+#ifdef HAVE_NETINET_IP_H
```

Comment:
`ip.h` provides `IPTOS_LOWDELAY`so how this setting is available on ESP32?

Reply:
in `lwip/sockets.h` which comes from `netdb.h` (above)